src/test/java/seedu/address/testutil/TypicalTasks.java

```diff
@@ -0,0 +1,33 @@
+package seedu.address.testutil;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import seedu.address.model.AddressBook;
+import seedu.address.model.task.Task;
+
+public class TypicalTasks {
+
+    public static final Task TASK1 = new TaskBuilder().withLabel("Buy red button")
+            .withDate("18th of September 2021").build();
```

Comment:
Is the absence of `withIsDone(false)` intended?

Reply:
Yes, this is just to make use of the default case, thanks for pointing out!